tools/jupyter_job/run_jupyter_job.xml

```diff
@@ -0,0 +1,80 @@
+<tool id="run_jupyter_job" name="Run long running jupyterlab job" version="0.0.1">
+    <description>on GPUs</description>
+    <requirements>
```

Comment:
I thought this needs to run in a Docker container? We should make this explicit, as we don't want to run this in conda, for security reasons?

Reply:
ping @anuprulez

Reply:
This dynamic code execution is the same as executing code written in any other interactive tool. The execution command from this tool can be sent to any secluded VM for running as far as I understand.

Reply:
Yes, but this tool is way more insecure than any other tool, left? So making sure it runs in Docker and not by accident in Conda is making it a little bit more secure?

Reply:
I agree, having a docker container running this tool will automatically enable it to run in a secluded environment.

Reply:
Moving to an explicit container requirement does not in any way guarantee this is safe. The right way *right now* is to make it an interactive tool. The medium term solution is a tag that means we require containerized execution and pick a destination that can run it.

Reply:
Current tool is supposed to be executed on jupyterlab notebook that is already an interactive tool. I am not sure if interactive tools interact with one another?

Reply:
No problem, they're just tools.